src/pybind/mgr/dashboard/controllers/nvmeof.py

```diff
@@ -367,7 +368,7 @@ def create(
                     rbd_pool_name=rbd_pool,
                     block_size=block_size,
                     create_image=create_image,
-                    size=rbd_image_size,
+                    size=rbd_image_size or size or 1024,
```

Comment:
With the change above:
```suggestion
                    size=rbd_image_size or size,
```